### PR TITLE
Add tfds notebook

### PR DESCRIPTION
**Pull request by rsepassi**
Reviewer, please test locally with GPU. I wasn't able to get CUDA working.

### CODE REVIEW

**Comment by alexbw** [comment]
This doesn't seem to run on public colabs. The runtime logs show this message:

```
2019-01-31 18:32:29.392004: E tensorflow/core/platform/cloud/curl_http_request.cc:596] The transmission of request 0xc16d180 (URI: http://metadata/computeMetadata/v1/instance/service-accounts/default/token) has been stuck at 0 of 0 bytes for 61 seconds and will be aborted. CURL timing information: lookup time: 0.005683 (No error), connect time: 0 (No error), pre-transfer time: 0 (No error), start-transfer time: 0 (No error)
```

I haven't been able to run the training, but it looks from the logs that were already printed in the checked in file that it's not training.

Also, please change the default runtime in this file to Python 3, because the pip install commands in this notebook are not compatible with Python 2.

**Comment by rsepassi** [comment]
Ah, thanks for finding that. Will fix and update the PR this coming week.

**Comment by rsepassi** [comment]
* Found and fixed the hanging.
* Model is now training (learning rate had to be lowered - same as the Pytorch one)
* Kernel is Py3

**Comment by alexbw** [comment]
Nice, all things fixed, lgtm!

**Comment by mattjj** [comment]
Awesome, thanks Ryan for this contribution, and thanks Alex for the review!

**Comment by rsepassi** [comment]
Woohoo! Thanks guys!
